addons/greatsql-cluster/values.yaml

```diff
@@ -0,0 +1,31 @@
+version: greatsql
+
+mode: standalone
+
+## @param replicas specify cluster replicas
+##
+replicas: 1
+
+## @param cpu
+##
+cpu: 0.5
```

Comment:
please adjust the cpu and memory values properly to make sure the cluster can run under such resource settings.